examples/core/cubemap/app.js

```diff
@@ -1,14 +1,20 @@
 /* global document */
-import {AnimationLoop, GL, TextureCube, Cube, Matrix4, radians} from 'luma.gl';
+import {
+  AnimationLoop, GL, TextureCube, Cube, Matrix4, radians,
+  resetParameters, setParameters
+} from 'luma.gl';
 
 const animationLoop = new AnimationLoop({
   onInitialize: ({gl, canvas}) => {
     addControls(canvas);
 
-    gl.clearColor(0, 0, 0, 1);
-    gl.clearDepth(1);
-    gl.enable(GL.DEPTH_TEST);
-    gl.depthFunc(GL.LEQUAL);
+    resetParameters(gl);
```

Comment:
@shaojingli @1chandu TIL!

In light of this, it might be interesting to give a `reset` parameter to `AnimationLoop.start()`. If that parameter is true we would call `resetParameters` on the context before calling `onInitailize`.

The discussion is mostly whether `reset` should be `true` or `false` by default. Probably what the user would expect, but open to opinions.